README.md

```diff
@@ -385,3 +385,16 @@ func main() {
     // ...
 }
 ```
+
+### Errata
```

Comment:
This would technically be addendum, not errata; it's not correcting anything, just adding.

Reply:
Fixed